Makefile

```diff
@@ -1,4 +1,6 @@
-OS=$(shell ./deps/readies/bin/platform --osnick)
+OS?=$(shell ./deps/readies/bin/platform --dist)
+OS_VER?=$(shell ./deps/readies/bin/platform --version)
+OS_NICK?=$(shell ./deps/readies/bin/platform --osnick)
 GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
```

Comment:
Various issues with how this reverses under tags.  Take a look at readies/mk/docker.rules#19-31

Reply:
Not follow, can you be more specific on what you advice to change it?